spec/fixtures/templates/rb/cfndsl/sample-ctp-repeated.rb

```diff
@@ -10,7 +10,7 @@
   Output(:One,FnBase64( Ref("One")))
 
   EC2_Instance(:MyInstance) {
-    DisableApiTermination external_parameters["DisableApiTermination"]
+    DisableApiTermination external_parameters.fetch(:DisableApiTermination, "false")
```

Comment:
In V1, `cfndsl` doesn't allow compile time parameters to be missing. We need to provide a default.